tests/test_github_compliance_alerter.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
```

Comment:
It should be possible to merge `test_github_compliance_alerter.py` and `test_s3_compliance_alerter.py` otherwise we have two test files that have a majority of code being duplicated.

Reply:
Sorted, if you could take a look at PR again please @achatain